test/Orleans.CodeGenerator.Tests/Orleans.CodeGenerator.Tests.csproj

```diff
@@ -0,0 +1,47 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <OrleansBuildTimeCodeGen>true</OrleansBuildTimeCodeGen>
+    <TargetFrameworks>$(TestTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <CentralPackageTransitivePinningEnabled>false</CentralPackageTransitivePinningEnabled>
+    <xUnitVersion>2.9.3</xUnitVersion>
```

Comment:
due to the dependency on [xunit.extensibility.execution](https://www.nuget.org/packages/xunit.extensibility.execution/) see https://www.nuget.org/packages/Verify.Xunit/28.15.0#dependencies-body-tab

could we upgrade the xunit versions in this repo? (maybe separate PR)

Reply:
I think a separate PR would be better, are we still supporting netcoreapp3.1? There are some conditions there, we'll discuss this in the new PR for xunit upgrades